Ptt/Flows/LoginFlow/Controllers/LoginSubView.swift

```diff
@@ -24,6 +24,13 @@ extension LoginViewController {
             ]
         }
         NSLayoutConstraint.activate(constraints)
+
+#if DEVELOP // Disable btnUserAgreement and btnForget, for now
+#else
+        btnUserAgreement.isHidden = true
+        vLine.isHidden = true
+        btnForget.isHidden = true
+#endif
     }
```

Comment:
Made modifications for adjusting scrollview visible area on hidden buttons: https://github.com/Ptt-official-app/Ptt-iOS/commit/008e293aa57c04fc0799de9c8f09aa81adcb8f6c